magicctapipe/scripts/lst1_magic/lst1_magic_stereo_reco.py

```diff
@@ -24,6 +24,10 @@
 (--output-dir dl1_stereo)
 (--config-file config.yaml)
 (--magic-only)
+
+Broader usage:
```

Comment:
this also can be deleted now - it will be automatically added with the other PR

Reply:
Done